### PR TITLE
Force to lowercase the ticket value for the `assigned_to` field

### LINKED ISSUE #740: Force lowercase on all ticket assignments
Currently `TicketConference.assigned_to` preserves the casing, it should default to lowercasing all the emails (easier to for exact matches in the backend code)

### PR DESCRIPTION
**Pull request by aramboi**
Add tests for the forced lowercase and strip sorrounding spaces.
Refactor test_models for pytest.
Optimize test collection and improve tests speed by 25s.
Refactor the way signals are imported in the p3 Django app.
Minor pep8 cleanup and readability fixes.

Fixes #740

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -103,13 +103,10 @@ clean: clean-docker clean-py
 clean-docker:
 	docker-compose down -t 60
 	docker system prune -f
-	docker volume prune -f
```
Comment (aramboi) [inline Makefile:106]: This was potentially destructive to peoples other container volumes. Also, we don't use volumes. 😅

File: p3/apps.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from django.apps import AppConfig
+
+
+class P3Config(AppConfig):
+    name = 'p3'
+
+    def ready(self):
+        import p3.listeners  # noqa
```
Comment (aramboi) [inline p3/apps.py:8]: This is the recommended way to import signals into django. Previously, this was imported at the end of models.py.

File: p3/models.py [hunk 3]
```diff
@@ -32,8 +33,8 @@ class SpeakerConference(models.Model):
 class TicketConferenceQuerySet(models.QuerySet):
     def available(self, user, conference=None):
         """
-        restituisce il qs con i biglietti disponibili per l'utente;
-        disponibili significa comprati dall'utente o assegnati a lui.
+        Returns the qs with the tickets available to the user;
+        available means bought by the user or assigned to him.
```
Comment (aramboi) [inline p3/models.py:37]: I google translated this ... seems legit.

File: p3/models.py [hunk 4]
```diff
@@ -95,6 +100,12 @@ class TicketConference(models.Model):
     def __str__(self):
         return "for ticket: %s" % self.ticket
 
+    def save(self, *args, **kwargs):
+        if self.assigned_to.strip():
+            self.assigned_to = self.assigned_to.lower().strip()
```
Comment (aramboi) [inline p3/models.py:105]: This is the actual logic that targets the ticket 😆 
The rest is cleanups and optimizations.

File: setup.cfg [hunk 5]
```diff
@@ -41,6 +42,14 @@ exclude_lines =
 env =
     DJANGO_SETTINGS_MODULE=pycon.test_settings
 
+testpaths =
+    ./tests
+    ./common/tests
```
Comment (aramboi) [inline setup.cfg:47]: This speeded up tests about 25s because the collection takes a lot less now.

File: setup.cfg [hunk 6]
```diff
@@ -50,4 +59,4 @@ addopts =
     --cov-config .coveragerc --cov=./ --cov-report=term
 
 filterwarnings =
-    ignore::django.utils.deprecation.RemovedInDjango20Warning
```
Comment (aramboi) [inline setup.cfg:53]: We will see all 2.0 warning now during tests ... and that is a good thing.

**Review by switowski — APPROVED** [review]

File: setup.cfg (hunk 6 above)
Comment (umgelurgel) [reply setup.cfg:53]: And shouldn't be for long, either :)

**Review by umgelurgel — APPROVED** [review]
Looks great! Thanks for addressing this and cleaning a few things up on the way :)

**Comment by malemburg** [comment]
Please don't use volume prune from the Makefile. This should
never be run on the server, not even accidentally, since we
do use them there. Anything which removes such volumes on the
server needs to be done explicitly and intentionally.

Thanks,
-- 
Marc-Andre Lemburg
EuroPython Society Chair
http://www.europython-society.org/
http://www.malemburg.com/

**Pull request merged**